python/cugraph-dgl/cugraph_dgl/utils/test_utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Also, same question about type hints, since if this module is meant to be user-facing then it should have type hints (if we're committing to those in this PR).

Reply:
This is not a user facing module so not adding type-hints here . Will add in a followup PR. Will file an issue to track it

https://github.com/rapidsai/cugraph/issues/2922